pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -573,7 +573,7 @@ func deletionStartsWithFinalizer(oldObj interface{}, newAccessor metav1.Object,
 	}
 	oldAccessor, err := meta.Accessor(oldObj)
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("cannot access oldObj: %v", err))
+		utilruntime.HandleErrorWithLogger(logger, err, "Cannot access oldObj", "oldObj", oldObj)
```

Comment:
This is an odd error message. I think this can be enhanced to `"Cannot access meta data", "oldObj", oldObj)`.